### PR TITLE
Wtg24 show the status of fetching data from the backend

### PR DESCRIPTION
**Pull request by niksuu**

### CODE REVIEW

File: src/app/route/route-list/route-list.component.ts [hunk 1]
```diff
@@ -138,28 +140,48 @@ export class RouteListComponent implements OnInit{
   }
 
   getRoutes() {
+    this.isLoading = true; // Ustawienie flagi ładowania na true
+    this.isError = false;  // Resetowanie błędu przed nową próbą pobrania danych
```
Comment (Kondziow) [inline src/app/route/route-list/route-list.component.ts:144]: Please remove these comments or at least translate them into english

File: src/proxy.conf.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "/api": {
-    "target": "http://localhost:8090",
+    "target": "http://localhost:18120",
```
Comment (Kondziow) [inline src/proxy.conf.json:3]: I don't think it's correct, it should remain the "8090" as it was

**Review by Kondziow — CHANGES_REQUESTED** [review]

File: src/proxy.conf.json (hunk 2 above)
Comment (Drillllll) [reply src/proxy.conf.json:3]: yop, just don't include it in the commits (I do it like that)

File: src/app/route/route-list/route-list.component.ts (hunk 1 above)
Comment (Drillllll) [reply src/app/route/route-list/route-list.component.ts:144]: deja vu

**Pull request merged**